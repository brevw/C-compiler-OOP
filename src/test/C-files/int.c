#include "minic-stdlib.h"

int main() {
    int i = 2;
    print_i(i);
}
