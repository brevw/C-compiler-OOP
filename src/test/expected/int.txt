INCLUDE
STRING_LITERAL(minic-stdlib.h)
INT
IDENTIFIER(main)
LPAR
RPAR
LBRA
INT
IDENTIFIER(i)
ASSIGN
INT_LITERAL(2)
SC
IDENTIFIER(print_i)
LPAR
IDENTIFIER(i)
RPAR
SC
RBRA
Lexing: pass
